src/sentry/utils/samples.py

```diff
@@ -139,7 +139,7 @@ def load_data(platform, default=None, sample_name=None):
         return
 
     data = CanonicalKeyDict(data)
-    if platform in ('csp', 'hkpk', 'expectct', 'expectstaple'):
+    if platform in ('transaction', 'csp', 'hkpk', 'expectct', 'expectstaple'):
```

Comment:
Why is this change needed ? It was not here in the previous version if I remember correctly

Reply:
Without this the sample transaction event gets data added to the `extra`, `modules` and `request` interfaces which are generally not going to be present on a transaction event payload.

Reply:
The variable name `platform` is an awkward name as it is used to locate a sample/fixture file in `src/sentry/data/samples`. The files there are a mixture of both sample SDK events and special scenarios like `empty-exception` or `python-omittedbody`.